code_examples/sdk_examples/src/dapp/verifier/verifyCredentialMessage.ts

```diff
@@ -64,9 +64,13 @@ export async function main({ session, verifierKeys }: Param) {
       attestationChain,
       credential.rootHash
     )
+    if (attestation.revoked) {
+      console.log("Credential has been revoked and hence it's not valid.")
+      return
+    }
     if (isTrustedAttester(attestation.owner)) {
       console.log(
-        'The claim is valid. Claimers email:',
+        "The claim is valid. Claimer's email:",
```

Comment:
Single quote `yarn fix`

Reply:
That's actually what `yarn fix` produced 😄